tests/cluecode/test_copyrights.py

```diff
@@ -33,7 +33,7 @@
 
 
 import pytest
-pytestmark = pytest.mark.scanslow
```

Comment:
You need to keep this `pytestmark = pytest.mark.scanslow` please... these tests are slow and running only when the `--test-suite=all` option is used with pytest. So do not remove this

Reply:
How do I marked that it has been ported?

Reply:
You need to add more than one marker there them with this:
`pytestmark = [pytest.mark.scanpy3, pytest.mark.scanslow]`

And you likely will need to have an extra test suite run for the slow Py3 tests in the pipeliens definition